.travis.yml

```diff
@@ -42,7 +42,7 @@ install:
   # Install gox
   - go get github.com/gz-c/gox
   # Install dependences for building wallet
-  - if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_PULL_REQUEST" == false ]]; then sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils && nvm install 8; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_PULL_REQUEST" == false ]]; then sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils libudev-dev && nvm install 8; fi
```

Comment:
this might require installing both `libudev-dev` and `libusb-1.0-0-dev` packages.